modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs

```diff
@@ -230,6 +232,11 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
         /// </summary>
         public int? MaxMessageSize => LegacyCliModel.MaxMessageSize;
 
+        /// <summary>
+        /// The Maximum (IoT D2C) messages
```

Comment:
Please add in the summary that this is message buffer size.

Reply:
updated XML comment